code/go/internal/validator/test/packages/bad_deploy_variants/_dev/deploy/variants.yml

```diff
@@ -0,0 +1,2 @@
+variants:
+  - FOO: bar
```

Comment:
For clarity you can leave a comment here what's wrong with this file (missing `default`, misused array).

Reply:
Added in https://github.com/elastic/package-spec/pull/45/commits/790a8e3c14befff47c5a5b620c0fb02434ee7d0e.